app/views/admin_merchants/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>New Admin Merchant</h1>
+<div id=create_admin_merchant>
+  <%= form_with url: "/admin/merchants", method: :post, local: true do |form| %>
```

Comment:
this will need to be changed to the form_with model style in the refactor

Reply:
agree